below/cgroupfs/src/types.rs

```diff
@@ -34,6 +34,12 @@ pub struct CpuMax {
     pub period_usec: u64,
 }
 
+#[derive(Default, Clone, PartialEq, Eq, Debug, Serialize, Deserialize)]
+pub struct PidsCurrent {
```

Comment:
This struct is not used anywhere in cgroupfs crate. Let's remove it.

Reply:
Thanks, fixed.